src/visibility.js

```diff
@@ -152,12 +160,12 @@ class Visibility extends BasePlugin {
     this._playerSizeBeforeFloating = this._state.shell.playerSize;
     Utils.Dom.addClassName(this._floatingContainer, FLOATING_ACTIVE_CLASS);
     Utils.Dom.addClassName(this._floatingPoster, FLOATING_POSTER_CLASS_SHOW);
-    Utils.Dom.setStyle(this._floatingContainer, 'height', `${this._floatingContainerHeight}px`);
+    Utils.Dom.setStyle(this._floatingContainer, 'height', `${this._getFloatingContainerHeight()}px`);
     Utils.Dom.setStyle(this._floatingContainer, 'width', this.config.width + 'px');
     Utils.Dom.setStyle(this._floatingContainer, 'margin', `${this.config.marginY}px ${this.config.marginX}px`);
     if (this.config.dismissible) {
-      const dismissibleContainerEl = Utils.Dom.getElementById(FLOATING_DISMISSIBLE_CONTAINER_ID);
-      this._floatingContainer.prepend(dismissibleContainerEl);
+      const dismissibleContainerEl = this._getDismissibleContainerEl();
+      dismissibleContainerEl && this._floatingContainer.prepend(dismissibleContainerEl);
```

Comment:
Same comment as in the other PR - i think an if would be more readable.

Reply:
ok